vnext/Shared/WinRTWebSocketResource.cpp

```diff
@@ -333,7 +333,7 @@ void WinRTWebSocketResource::OnMessageReceived(IWebSocket const& sender, IMessag
 
     if (m_readHandler)
     {
-      m_readHandler(response.length(), response);
+      m_readHandler(response.length(), response, args.MessageType() != SocketMessageType::Utf8);
```

Comment:
nit: should we use:

args.MessageType() == SocketMessageType::Binary

in case later types are added for other text encodings?

see:
[https://docs.microsoft.com/en-us/uwp/api/Windows.Networking.Sockets.SocketMessageType?view=winrt-19041](https://docs.microsoft.com/en-us/uwp/api/Windows.Networking.Sockets.SocketMessageType?view=winrt-19041)

Reply:
I believe the WebSocket spec does handle only binary/octet or utf8/ascii messages.

That said, we can play it safe by just comparing against `SocketMessageType::Binary`.